tests/networks.spec.ts

```diff
@@ -4,148 +4,378 @@ import {
+    network = `${browserName.substring(0, 2)}-${network}`;
+    const page = await browser.newPage();
+    await createNetwork(page, network, "physical");
+    await page.close();
+  });
+
+  test.afterAll(async ({ browser }) => {
+    const page = await browser.newPage();
+    await deleteNetwork(page, network);
+    await page.close();
+  });
+
+  test("configure physical network DNS nameservers", async ({ page }) => {
+    const DNS_NAMESERVERS = "32.152.61.210";
+
+    await prepareNetworkTabEdit(page, "DNS", network);
+
+    await setTextarea(page, "DNS nameservers", DNS_NAMESERVERS);
+  });
+
+  test("set physical network IPv4 routes and gateway", async ({ page }) => {
+    const options = ["true", "false"];
+    const OVN_RANGES = "10.141.1.100-10.141.1.110";
+    const IPV4_ROUTES = "10.141.1.1";
+
+    await prepareNetworkTabEdit(page, "IPv4", network);
+
+    await setTextarea(page, "IPv4 OVN ranges", OVN_RANGES);
+    await setTextarea(page, "IPv4 gateway", NETWORK_CONSTANTS.IPV4_ADDRESS);
+    await activateOverride(page, "IPv4 routes Additional IPv4");
+    await page.getByLabel("IPv4 routes").fill(IPV4_ROUTES);
+    for (const option of options) {
+      await setOption(page, "IPv4 routes anycast", option);
+    }
+  });
+
+  test("configure physical network IPv6 routes and gateway", async ({
+    page,
+  }) => {
+    const options = ["true", "false"];
+    const OVN_RANGES = "2001:db8::100-2001:db8::150";
+    const IPV6_GATEWAY = "2001:db8::100";
+
+    await prepareNetworkTabEdit(page, "IPv6", network);
+
+    await setTextarea(page, "IPv6 OVN ranges", OVN_RANGES);
+    await setTextarea(page, "IPv6 gateway", IPV6_GATEWAY);
+    await activateOverride(page, "IPv6 routes Additional IPv6");
+    await page.getByLabel("IPv6 routes").fill(IPV6_GATEWAY);
+    for (const option of options) {
+      await setOption(page, "IPv6 routes anycast", option);
+    }
+  });
+
+  test("set physical network OVN ingress mode", async ({ page }) => {
+    const options = ["l2proxy", "routed"];
+
+    await prepareNetworkTabEdit(page, "OVN", network);
+
+    for (const option of options) {
+      await setOption(page, "OVN ingress mode", option);
+    }
+  });
 });
 
-test("network forwards", async ({ page }) => {
-  await createNetworkForward(page, network);
+test.describe("OVN type", () => {
+  test.beforeAll(async ({ browser, browserName }) => {
+    // network names can only be 15 characters long
+    network = `${browserName.substring(0, 2)}-${network}`;
+    const page = await browser.newPage();
+    await page.goto("/ui/settings");
+
+    const parent = page.getByRole("row").filter({
+      hasText: "network.ovn.northbound_connection",
+    });
+    await parent.getByRole("button").click();
+
+    initialNorthboundNetworkValue = await parent
+      .getByRole("textbox")
+      .inputValue();
+
+    await parent.getByRole("textbox").fill("foo");
+
+    await page
+      .getByRole("button", {
+        name: /save/i,
+      })
+      .click();
```

Comment:
```suggestion
  await visitServerSettings(page);
  await updateSetting(page, "network.ovn.northbound_connection", "text", "foo");
```
We can use an existing abstraction for this. Also for the reset in after all below.

Reply:
I forgot: We also need `getServerSettingValue` for the initial value.